src/Application/Features/Teams/Queries/GetTeam/GetTeamQuery.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.EntityFrameworkCore;
+using SSW.CleanArchitecture.Application.Common.Interfaces;
+using SSW.CleanArchitecture.Domain.Teams;
+
+namespace SSW.CleanArchitecture.Application.Features.Teams.Queries.GetTeam;
+
+public record GetTeamQuery(Guid TeamId) : IRequest<TeamDto?>;
+
+public sealed class GetAllTeamsQueryHandler(IApplicationDbContext dbContext) : IRequestHandler<GetTeamQuery, TeamDto?>
+{
+    public async Task<TeamDto?> Handle(
+        GetTeamQuery request,
+        CancellationToken cancellationToken)
+    {
+        var teamId = new TeamId(request.TeamId);
+
+        var team = await dbContext.Teams
+            .Where(t => t.Id == teamId)
+            .Select(t => new TeamDto
+            {
```

Comment:
No Automapper?

Reply:
I probably should have.  Will aim to tidy this up later.